pkg/cmd/server/api/helpers.go

```diff
@@ -73,9 +86,9 @@ func (fl *FeatureList) Delete(items ...string) {
 // Has returns true if given feature exists in feature list. The check is
 // case-insensitive.
 func (fl FeatureList) Has(feature string) bool {
-	lowerCased := strings.ToLower(feature)
+	normalized := NormalizeOpenShiftFeature(feature)
```

Comment:
this changes behavior from just lowercasing to limiting to the set of known flags... and also means that if the feature we're checking, and a feature in the FeatureList are both unknown, they will match (""=="")... need to rework this